test/integration/pack/test_pack.py

```diff
@@ -598,8 +598,8 @@ def test_invalid_base_build_dockerfile(cartridge_cmd, project_without_dependenci
 
         rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
         assert rc == 1
-        assert 'Invalid base build Dockerfile' in output
-        assert 'base image must be centos:8' in output
+        assert 'Failed to build base image' in output
+        assert 'The centos:8 image is expected to be used' in output
```

Comment:
I think it would be better to check that using `FROM some-cute-name-for-centos:8` completes successfully, but shows a warning.

Using `alexeymrvz/my-custom-centos8` isn't a good solution for tests, you can create such image locally (as a fixture) using python docker module.

Reply:
The same for `test_pack_docker.py::test_custom_base_runtime_dockerfile `